docs/develop/01_sdk/02_cookbook/03_account_linking/01_link.md

```diff
@@ -4,20 +4,73 @@ title: Link an Account to a KILT DID
+import SubAccLink from '!!raw-loader!@site/code_examples/sdk_examples/src/core_features/linking/01_sub_link.ts';
+import EthAccLink from '!!raw-loader!@site/code_examples/sdk_examples/src/core_features/linking/01_eth_link.ts';
+import EthWeb3AccLink from '!!raw-loader!@site/code_examples/sdk_examples/src/core_features/linking/01_eth_link_web3js.ts';
+import SenderLink from '!!raw-loader!@site/code_examples/sdk_examples/src/core_features/linking/02_sender_link.ts';
 
-An account can be linked to a DID in one of two ways:
+Sometimes there is the need to link a DID to an account publicly.
+The link makes it possible to lookup a DID for an account.
+The other directions is also possible.
+With a DID you can lookup a list of linked account.
 
-1. The account to be linked generates a signature over the payload `<Bytes>(SubmitterDid, BlockNumberExpiration)</Bytes>`, which is the default encoding of the PolkadotJS extension:
+Linking accounts can be useful when your account should have an identity.
+E.g. as a collator, you might want to provide some public information so that delegator can better decide who earned their stake.
+
+An account can be linked to a DID in one of two ways.
+Either the account that sends the transaction links itself to the DID or the sender is unrelated to the DID and a third account is linked.
+In that case a challenge needs to be signed with the third account, to proof ownership.
+
+The second option is useful in cases where the account that should be linked doesn't own KILT tokens or the transaction is paid for by a third party.
+
+:::warning Don't use linked accounts for asset transfers
+
+Don't use these linked accounts for asset transfers.
+Since these accounts are not limited to KILT accounts, but can be used on any chain, the recipient might not be able to access the transferred asset on other chains.
+When a link to an account on a different substrate chain is created, this account might only be usable on this specific chain.
+
+If you want transfer assets to a DID have a look at [the asset transfer service endpoint](https://github.com/KILTprotocol/spec-KiltTransferAssetRecipientV1).
+
+:::
+
+## Linking the sender to a DID
+
+Link the sender of the transaction to the DID.
+The sender will provide the deposit and pay the fees.
+He will also be linked to the DID.
 
 <TsJsBlock>
-  {DidLink}
+  {SenderLink}
 </TsJsBlock>
 
-2. The account submitting the linking transaction is linked to the DID authorizing such transaction:
+## Linking an account to a DID
 
-<TsJsBlock>
-  {AccountLink}
-</TsJsBlock>
\ No newline at end of file
+Link another account to the DID.
+The sender will provide the deposit and pay the fees, but will not be linked to the DID in any way.
+The account that should be linked must sign a challenge to proof that the account agrees to be linked.
+
+The proof contains the DID that the account will be linked to and an expiration date (as a blocknumber), to prevent replay attacks.
```

Comment:
```suggestion
The proof contains the DID that the account will be linked to and an expiration date (in terms of blocks), to prevent replay attacks.
```